sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/metadata.yaml

```diff
@@ -1,12 +1,18 @@
----
 friendly_name: Urlbar Clients Daily
-description: Daily per-client metrics related to urlbar usage
+description: |-
+  Daily per-client metrics related to urlbar usage
 owners:
-  - akomar@mozilla.com
-  - tbrooks@mozilla.com
+- akomar@mozilla.com
+- tbrooks@mozilla.com
 labels:
   application: firefox
   incremental: true
   schedule: daily
 scheduling:
   dag_name: bqetl_urlbar
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+  clustering: null
```

Comment:
Would it be worth specifying clustering fields?

Reply:
Yes, this is likely an oversight. The table contains `normalized_channel`, which we usually do use as a clustering field when available.

Reply:
🙇